TLSharp.Core/Network/TcpTransport.cs

```diff
@@ -12,7 +12,7 @@ public class TcpTransport : IDisposable
         private readonly TcpClient _tcpClient;
         private int sendCounter = 0;
 
-        public TcpTransport(string address, int port, TcpClientConnectionHandler handler = null)
+        public TcpTransport(string address, int port, bool keepAlive = false, TcpClientConnectionHandler handler = null)
```

Comment:
github cannot push code from you by mistake :) you pushed it to the wrong branch ;)